src/Bootstraps/DiscoveryBootstrap.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Bootstraps;
+
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use ReflectionClass;
+use SplFileInfo;
+use Tempest\AppConfig;
+use Tempest\Container\Container;
+use Tempest\Discovery\Discovery;
+use Throwable;
+
+final readonly class DiscoveryBootstrap implements Bootstrap
```

Comment:
I moved the `initDiscovery` method to a bootstrapper as well

Reply:
It was right in front of me the whole time. 😫